content/introduction/overview.md

```diff
@@ -47,36 +51,34 @@ What about that link to the Aardvark page above? It looked a little unusual:
 
-It’s important to remember in all of these situations that it’s the address or link that doesn’t change. The IPFS network is a participatory and collaborative one — if nobody has the content identified by a given address, it won’t be available. On the other hand, content can’t be removed from IPFS so long as *someone* is interested enough to make it available, whether or not that person is the original author.
+It’s important to remember in all of these situations using IPFS is participatory and collaborative. If nobody has the content identified by a given address available for others to access, you won't be able to get it. On the other hand, content can’t be removed from IPFS as long as *someone* is interested enough to make it available, whether that person is the original author or not.
 
 
 ## It’s all about possession and participation.
 
-While there is lots of complex technology in IPFS, the fundamental ideas are about changing how networks of people and computers communicate: while today’s world-wide-web is structured on *ownership* and *access*, where you get files from whoever owns them and they choose to grant you access, IPFS is based on the ideas of *possession* and *participation*, where many people have each others’ files and *participate* in making them available.
+While there's lots of complex technology in IPFS, the fundamental ideas are about changing how networks of people and computers communicate. Today’s world-wide-web is structured on *ownership* and *access*, where you get files from whoever owns them and they choose to grant you access. IPFS is based on the ideas of *possession* and *participation*, where many people have each others’ files and *participate* in making them available.
 
-That means IPFS only works well when people are actively participating. If you share a file on your computer, but then turn your computer off, other people won’t be able to access it from you. But if you or others make sure a copy of that file is stored on another computer running IPFS, it will be available more often. By default, your computer participates in sharing a file with others for a limited time after you’ve downloaded it with IPFS. You can *pin* a file to save and permanently share it.
+That means IPFS only works well when people are actively participating. If you share a file on your computer, but then turn your computer off, other people won’t be able to access it from you. But if you or others make sure a copy of that file is stored on another computer running IPFS, it will be available more often. By default, your computer participates in sharing a file with others for a limited time after you’ve downloaded it with IPFS. You can *pin* a file to save and permanently share it (and you should read the [concept guide on pinning]({{<relref "guides/concepts/mfs.md">}}) to learn more about that!).
```

Comment:
"If you use your computer to share files using IPFS, but then you turn your computer off, other people won't be able to get those files from you any more. But if you or others make sure that copies of those files are stored on more than one computer that's powered on and running IPFS, those files will be more reliably available to other IPFS users who want them. This happens to some extent automatically -- because by default, your computer shares a file with others for a limited time after you've downloaded it using IPFS -- but you can also make file availability more permanent by *pinning* it to save and permanently share it. (You can learn more about this in the [concept guide on pinning]({{<relref "guides/concepts/mfs.md">}})!)

Reply:
This is much better! I took another rev on your start, here it be:

```That means IPFS only works well when people are actively participating. If you use your computer to share files using IPFS, but then you turn your computer off, other people won't be able to get those files from you anymore. But if you or others make sure that copies of those files are stored on more than one computer that's powered on and running IPFS, those files will be more reliably available to other IPFS users who want them. This happens to some extent automatically: by default, your computer shares a file with others for a limited time after you've downloaded it using IPFS. You can also make content available more permanently by *pinning* it, which saves it to your computer and makes it available on your IPFS network until you decide to *unpin* it. (You can learn more about this in the [concept guide on pinning]({{<relref "guides/concepts/mfs.md">}})!)```

Reply:
Perfect. Thanks for including info on un-pinning, I forgot that, and it's important.

Reply:
Teamwork! 🚢